environment.yml

```diff
@@ -1,4 +1,4 @@
-name: latent_space_visualizer
+name: latent_space_builder
```

Comment:
should we generate an compSPI environment instead?

Reply:
If we were to create an `environment.yml` file for setting up the `compSPI` environment, then we may need to store that file in a central location that is accessible to anyone who pulls the `LatentSpaceVisualizer` or `LatentSpaceBuilder` repositories.

One possible location would be the Anaconda Cloud.

After following the instructions in the first link pasted below, I was able to upload and download an `environment.yml` file to and from the Anaconda Cloud. 

I have attached the link below that contains the uploaded `environment.yml` file for your review.

Perhaps this step of downloading the `environment.yml` file from the Anaconda Cloud can be included in the `README.md` for the `LatentSpaceVisualizer` or `LatentSpaceBuilder` repositories. After the user downloads the `environment.yml` file, he can then proceed to creating the compSPI environment using the downloaded file.

The prerequisite for us to upload the `environment.yml` file so that it is accessible to the user is an Anaconda Cloud account. Creating an Anaconda Cloud user account is free. If a user account is not sufficient for our needs, creating an organization account is also possible by following the third link pasted below.

Link to the instructions for downloading/uploading an `environment.yml` file: [https://docs.anaconda.com/anaconda-cloud/user-guide/tasks/work-with-environments/](https://docs.anaconda.com/anaconda-cloud/user-guide/tasks/work-with-environments/)

Link to the `environment.yml` that I uploaded to the Anaconda Cloud: [https://anaconda.org/rdeeban/compSPI/files](https://anaconda.org/rdeeban/compSPI/files)

Link to creating an organization in Anaconda Cloud: [https://docs.anaconda.com/anaconda-cloud/user-guide/tasks/work-with-organizations/#creating-an-organization](https://docs.anaconda.com/anaconda-cloud/user-guide/tasks/work-with-organizations/#creating-an-organization)

Reply:
I like the idea of an Anaconda Cloud account for the compSPI organization! Creating one now :D

Reply:
I have created a discussion on that topic here: https://github.com/orgs/compSPI/teams/developers

Reply:
Thanks @rdeeban ! The answer was: `conda env create compSPI/compSPI`